src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Listeners/QueueListener.cs

```diff
@@ -166,6 +170,131 @@ public void Dispose()
         }
 
         public async Task<TaskSeriesCommandResult> ExecuteAsync(CancellationToken cancellationToken)
+        {
+            return _concurrencyManager.Enabled ? 
```

Comment:
I noticed you aren't checking for a null `ConcurrencyManager` anywhere from the point at which it gets injected to here. Should there be a null check here or somewhere further up the stack? (btw, it would be great to consider enabling C# nullable reference types in this codebase. If nothing else, it would make the intent more clear).

Reply:
Yes, ctor should check for null.